tests/publiccloud/transfer_repos.pm

```diff
@@ -31,7 +31,8 @@ sub run {
     if (get_var('QAM_PUBLICCLOUD_SKIP_DOWNLOAD') == 1) {
         record_info('Skip download', 'Skipping download triggered by setting (QAM_PUBLICCLOUD_SKIP_DOWNLOAD = 1)');
     } else {
-        assert_script_run("du -sh ~/repos");
+        assert_script_run('du -sch ~/repos/download.suse.de/ibs/SUSE:/Maintenance:/*');
```

Comment:
is it related to terraform profile issue ?

Reply:
It is not. It is just a debug command.

Reply:
do you want to keep it ?

Reply:
Should I put it away or is this OK?

Reply:
It's gona. Please merge when CI is done :)